addons/loyalty/wizard/loyalty_generate_wizard.py

```diff
@@ -28,6 +28,7 @@ class LoyaltyGenerateWizard(models.TransientModel):
     valid_until = fields.Date()
     will_send_mail = fields.Boolean(compute='_compute_will_send_mail')
     confirmation_message = fields.Char(compute='_compute_confirmation_message')
+    description = fields.Text(string="Descripton", default='Gift for customer')
```

Comment:
No need to add default values may be placeholder is better and add default value if empty

This way user don't have to every time remove this values and add his preferred values